extensions/jms/src/test/resources/config/activemq/activemq-with-overrides.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:jmsn="http://www.mulesoft.org/schema/mule/jmsn"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/jmsn http://www.mulesoft.org/schema/mule/jmsn/current/mule-jmsn.xsd">
+
+    <jmsn:config name="config">
+        <jmsn:active-mq-connection clientId="myConsumerClient">
+            <jmsn:factory-configuration brokerUrl="vm://localhost?broker.useJmx=false"
```

Comment:
factory-configuration?. Why are those attributes not part of active-mq-connection?